<!-- -->
<launch>

<node name="cppPublisher" pkg ="pub_sub_rate" type="cppPublisher" respawn="false" output="screen"/>
<node name="cppSubscriber" pkg ="pub_sub_rate" type="cppSubscriber" respawn="false" output="screen"/>

<node name="rate_plot"
        pkg="rqt_plot"
        type="rqt_plot"
        args="/CppSubFreqRate /CppPublisherRate" />

<node name="rqt_reconfigure" pkg ="rqt_reconfigure" type="rqt_reconfigure" respawn="false" output="screen"/>

</launch>



